Dockerfile

```diff
@@ -55,21 +68,48 @@ ARG USER_NAME
 ARG HOME_DIR
 ARG SOURCE_DIR
 
+RUN \
+  apt-get update \
+  && apt-get -y install gettext-base \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
 # Create the group (use a high ID to attempt to avoid conflits).
-RUN groupadd -g $USER_ID $USER_NAME
+RUN groupadd --non-unique -g $USER_ID $USER_NAME
 
 # Create the user (use a high ID to attempt to avoid conflits).
-RUN useradd -d $HOME_DIR -m -u $USER_ID -g $USER_ID $USER_NAME
+RUN useradd --non-unique -d $HOME_DIR -m -u $USER_ID -g $USER_ID $USER_NAME
 
 # Login as user.
 USER $USER_NAME
 
 # Set deployment directory.
 WORKDIR $HOME_DIR
 
+
 # Copy over the built artifact and library from the maven image.
 COPY --from=maven $SOURCE_DIR/target/ROOT.jar ./sage.jar
 COPY --from=maven $SOURCE_DIR/target/libs ./libs
 
+# Copy app config.
+COPY build/appConfig.js.template /usr/local/app/templates/appConfig.js.template
+
+# Copy of docker entrypoint to user local binary directory.
+COPY build/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
+
+ENV AUTH_STRATEGY weaverAuth
+
+ENV STOMP_DEBUG false
+
+ENV AUTH_SERVICE_URL http://localhost:9001/auth
+ENV AVALON_URL avalon-pre.library.tamu.edu:443
+
+ENV APP_CONFIG_PATH=file:$HOME_DIR/appConfig.js
```

Comment:
I've had problems with `file:` not working.
I've had to change this to something more like:
```
ENV APP_CONFIG_PATH $HOME_DIR/appConfig.js
```

If the `appConfig.js` is placed in the same directory that `CMD ["java", "-jar", "./sage.jar"]` is run, then this can further be simplified to:
```
ENV APP_CONFIG_PATH ./appConfig.js
```
(I think maybe CAP could be updated as well.)

Reply:
Updated. Updated CAP as well.